openesdh-repo/src/main/java/dk/openesdh/repo/services/activities/CaseDocumentActivityBehaviour.java

```diff
@@ -69,4 +78,7 @@ public void onCreateVersion(QName classRef, NodeRef versionableNode,
         }
     }
 
+    private boolean docDoesntBelongToCase(NodeRef versionableNode) {
+        return !Optional.ofNullable(documentService.getCaseNodeRef(versionableNode)).isPresent();
```

Comment:
in my opinion this is to complicated to read for such trivial action  as == null :)